staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion.go

```diff
@@ -534,17 +549,56 @@ func (g *genConversion) convertibleOnlyWithinPackage(inType, outType *types.Type
 	return true
 }
 
-func (g *genConversion) Filter(c *generator.Context, t *types.Type) bool {
-	peerType := getPeerTypeFor(c, t, g.peerPackages)
-	if peerType == nil {
-		return false
-	}
-	if !g.convertibleOnlyWithinPackage(t, peerType) {
-		return false
+func getCustomFromTypes(t *types.Type) []types.Name {
+	comments := append(t.SecondClosestCommentLines, t.CommentLines...)
+	paths := extractCustomFromTag(comments)
+	result := []types.Name{}
+	for _, path := range paths {
+		items := strings.Split(path, ".")
+		if len(items) != 2 {
+			klog.Errorf("Unexpected k8s:conversion-gen:explicit-from tag: %s", path)
+			continue
+		}
+		switch {
+		case items[0] == "net/url" && items[1] == "Values":
+		default:
+			klog.Errorf("Not supported k8s:conversion-gen:explicit-from tag: %s", path)
```

Comment:
Do we not hard error on these?  Just write errors and accumulate?

Reply:
Good point - changed to Fatal.